lib/Common/Exceptions/Throw.cpp

```diff
@@ -77,6 +77,13 @@ namespace Js {
         ReportFatalException(NULL, E_FAIL, Fatal_Internal_Error, scenario);
     }
 
+    void Throw::FatalInternalGlobalizationError()
+    {
+        AssertMsg(false, "Failure in initializing Globalization library");
+        int scenario = 2;
```

Comment:
Do you know what these scenarios mean? I thought they were for bucketing but now I see that multiple different fatal errors use the same scenario...

Reply:
Yeah TBH I'm not sure. There's also the difference between the file that defines the `*_unrecoverable_*` functions (previously `*_fatal_*`) and the `Throw::` functions. Bucketing is done via callstack though so I think that's the important bit? @tcare do you know what the differences here are?